pkg/volume/plugins.go

```diff
@@ -483,7 +483,7 @@ func (pm *VolumePluginMgr) initProbedPlugin(probedPlugin VolumePlugin) error {
 	if err != nil {
 		return fmt.Errorf("Failed to load volume plugin %s, error: %s", name, err.Error())
 	}
-
+	pm.probedPlugins[name] = probedPlugin
```

Comment:
`pm.probedPlugins[name]` is filled in `pm.refreshProbedPlugins`, just after `initProbedPlugin` returns. Do I miss anything?

Can you please post some sequence of operations where `pm.probedPlugins[name] = probedPlugin` on this place is necessary?